patches/chrome-installer-linux-rpm-build.sh.patch

```diff
@@ -25,3 +25,13 @@ index 261faeaf26f34f2e89229579ae7933e23e4299e6..201aacd532ea70067279ec970134f3b2
    mv "$RPMBUILD_DIR/RPMS/$ARCHITECTURE/${PKGNAME}.${ARCHITECTURE}.rpm" \
       "${OUTPUTDIR}"
    # Make sure the package is world-readable, otherwise it causes problems when
+@@ -146,6 +150,9 @@ verify_channel() {
+     unstable|dev|alpha )
+       CHANNEL=unstable
```

Comment:
actually can we use unstable as nightly or do we need both?

Reply:
We need both - linux uses `unstable` and `dev`  both for `dev` channel.